.github/workflows/tests.yml

```diff
@@ -41,7 +41,7 @@ jobs:
   tests:
     # Will not run if the event is a PR to bump-meilisearch-v* (so a pre-release PR)
     # Will still run for each push to bump-meilisearch-v*
-    if: github.event_name != 'pull_request' || !startsWith(github.base_ref, 'bump-meilisearch-v')
+    if: (endsWith(github.base_ref, '-beta') && (github.event_name != 'pull_request' || !startsWith(github.base_ref, 'bump-meilisearch-v')))
```

Comment:
I'm not sure I understand the logic here.

Are you requiring that the branch ends with `-beta` in order for the normal tests to run?

Reply:
Shouln't it be this:

```
    if: (!endsWith(github.base_ref, '-beta') || github.event_name != 'pull_request' || !startsWith(github.base_ref, 'bump-meilisearch-v'))

```

?